benefits/enrollment/templates/enrollment/success.html

```diff
@@ -7,26 +7,16 @@
   {{ block.super |add:" enrollment-success" }}
 {% endblock classes %}
 
-{% block paragraph_content %}
-  {% if "logged-out" in page.classes %}
-    <img class="success-image" src="{% static "img/icon/happybus.svg" %}" alt="{% translate "core.icons.happybus" context "image alt text" %}">
-  {% else %}
-    <p>{% translate "enrollment.pages.success.p1" %}</p>
-    <p>{% translate "enrollment.pages.success.p2" %}</p>
-    <p>{% blocktranslate with help_link=help_link%}enrollment.pages.success.p3{{ help_link }}{% endblocktranslate %}</p>
-  {% endif %}
```

Comment:
Don't we still need this logged out view?

Reply:
Yeah thanks for this reminder, we talked about moving it out of the success view and I forgot to do that here.

Reply:
It didn't actually make sense to move this out of the success view and create a specific "logged out" view - since the user can log out from various places in the app, we can't send them to a final "you are logged out" _everytime_ they log out - we use the `session.origin()` to know where to send the user.

Reply:
See [6f0db24](https://github.com/cal-itp/benefits/pull/997/commits/6f0db241880135aa4b734b64776173dc483c54c0)